src/SignhostAPIClient/Rest/ErrorHandling/HttpResponseMessageErrorHandlingExtensions.cs

```diff
@@ -99,8 +99,6 @@ public static class HttpResponseMessageErrorHandlingExtensions
 					throw new SignhostRestApiClientException(
 						errorMessage);
 			}
-
-			System.Diagnostics.Debug.Fail("Should not be reached");
```

Comment:
Well yes that was the whole point this failure was here, in case we somehow introduce a bug making this line to be reachable so leave this in please.

Reply:
Commit dropped